### ISSUE TITLE
Option to check if account is created on custom closes

### ISSUE THREAD

**Issue by Matthewrbowker** [enwikipedia-acc/waca#250]
Related to #247, but in a more specific use case.  Specifically with the "Created (COI risk)" closure on the live config, there is no check to ensure that the account has actually been created.  Maybe this is something we can configure on a per-email basis?

**Comment by FastLizard4** [comment]
Your request is a little specific, so let me rephrase it and and ask if it's what you want - Are you asking for a feature that, if a request is closed as "created", whether using the standard creation button, a saved created close message, or a custom close as created, checks via the API if the account actually was created before allowing the request to close?

**Comment by Matthewrbowker** [comment]
Yes, you've hit the nail on the head.

**Comment by stwalkerster** [comment]
-`easy` because it's probably not easy to do this for custom close too

(Also, ping #232 )

**Comment by FastLizard4** [comment]
Partially fixed by #251.  251 will apply the API check to the non-standard templates that are marked as creation templates (such as the COI Risk template), but not to true custom closes.  Indeed, though I'll need to consult with @stwalkerster to confirm this, that will probably be something to aim for in newinternal instead of trying to shoehorn a fix together right now.

**Comment by FastLizard4** [comment]
Oops, didn't mean to close this.

**Comment by stwalkerster** [comment]
Yeah, it won't work for custom closes at the moment. I _think_ this may be fixed already in the newinternal branch, but I need to confirm it.

**Comment by stwalkerster** [comment]
Closing in preference of #689
